#message to represent a segment of the final path. That includes a dubins curve and a#stop point

Dubin_msg d_dubin
geometry_msgs/Vector3 stop_pt
